build.sbt

```diff
@@ -1,77 +1,57 @@
-import Dependencies._
+import codacy.libs._
 
 import scala.io.Source
 import scala.util.parsing.json.JSON
 
 name := """bitbucket-scala-client"""
 
-version := "1.9.0-SNAPSHOT"
```

Comment:
Isn't this needed for publishing locally?

Reply:
no, you will publish local always as `0.0.1-SNAPSHOT` in case you can change it with `set version in ThisBuild := "whatever"`